docs/02-app/01-building-your-application/02-data-fetching/03-patterns.mdx

```diff
@@ -43,7 +43,7 @@ With Server Components and [nested layouts](/docs/app/building-your-application/
 
 To learn more about Streaming and Suspense, see the [Loading UI](/docs/app/building-your-application/routing/loading-ui-and-streaming) and [Streaming and Suspense](/docs/app/building-your-application/routing/loading-ui-and-streaming#streaming-with-suspense) pages.
 
-### Parallel and Sequential Data Fetching
+## Parallel and Sequential Data Fetching
```

Comment:
We should be consistent with the casing. The above is changed to sentence case (which follows the style guide) 
```suggestion
## Parallel and sequential data fetching
```